sensor_msgs/Image image
---
openpose_ros/Person[] people

